collectors/python.d.plugin/python_modules/bases/FrameworkServices/MySQLService.py

```diff
@@ -50,12 +50,6 @@ def get_connection_properties(conf, extra_conf):
             elif conf.get('host'):
                 properties['host'] = conf['host']
                 properties['port'] = int(conf.get('port', 3306))
-            elif conf.get('my.cnf'):
-                if MySQLdb.__name__ == 'pymysql':
-                    # TODO: this is probablt wrong, it depends on version
-                    self.error('"my.cnf" parsing is not working for pymysql')
-                else:
-                    properties['read_default_file'] = conf['my.cnf']
```

Comment:
i think we shouldnt delete 

```python
elif conf.get('my.cnf'):
    properties['read_default_file'] = conf['my.cnf']
```

we want to get rid of 

```python
if MySQLdb.__name__ == 'pymysql':	
    # TODO: this is probablt wrong, it depends on version	
    self.error('"my.cnf" parsing is not working for pymysql')
```

Reply:
Updated the pull request